app/models/event.rb

```diff
@@ -171,8 +171,8 @@ def self.query_aggregations
     {
       sources: { terms: { field: 'source_id', size: 50, min_doc_count: 1 } },
       prefixes: { terms: { field: 'prefix', size: 50, min_doc_count: 1 } },
-      pairings: { terms: { field: 'subj.provider_id', size: 50, min_doc_count: 1 }, aggs: { citation_types: { terms: { field: 'obj.provider_id', size: 50, min_doc_count: 1 }, aggs: { "tota" => { sum: { field: 'total' }}}}} },
-      clients: { terms: { field: 'subj.provider_id', size: 50, min_doc_count: 1 }, aggs: { year_months: { date_histogram: { field: 'occurred_at', interval: 'month', min_doc_count: 1 }, aggs: { "total_by_year_month" => { sum: { field: 'total' }}}}} },
+      clients: { terms: { field: 'registrant_id', size: 50, min_doc_count: 1 }, aggs: { year_months: { date_histogram: { field: 'occurred_at', interval: 'month', min_doc_count: 1 }, aggs: { "total_by_year_month" => { sum: { field: 'total' }}}}} },
```

Comment:
Please don't use the term `clients`, as this has different meaning for Crossref and DataCite. `providers` is already a bad choice.

Reply:
fixed in 0d2ed3a